test/primer/hyperloglog_test.cpp

```diff
@@ -133,15 +133,15 @@ TEST(HyperLogLogTest, DISABLED_BasicParallelTest) {
 
   std::vector<std::thread> threads2;
   for (uint16_t k = 0; k < 10; k++) {
-    threads2.emplace_back(std::thread([&]() { obj.AddElem("Andy"); }));
-    threads2.emplace_back(std::thread([&]() { obj.AddElem("Connor"); }));
-    threads2.emplace_back(std::thread([&]() { obj.AddElem("J-How"); }));
-    threads2.emplace_back(std::thread([&]() { obj.AddElem("Kunle"); }));
-    threads2.emplace_back(std::thread([&]() { obj.AddElem("Lan"); }));
-    threads2.emplace_back(std::thread([&]() { obj.AddElem("Prashanth"); }));
-    threads2.emplace_back(std::thread([&]() { obj.AddElem("Will"); }));
-    threads2.emplace_back(std::thread([&]() { obj.AddElem("Yash"); }));
-    threads2.emplace_back(std::thread([&]() { obj.AddElem("Yuanxin"); }));
+    threads2.emplace_back(std::thread([&]() { obj.AddElem("Andy Pavlo"); }));
```

Comment:
Remove last names.

Reply:
Actually, I added it because I wanted the hash values to be distributed to different buckets. It was hitting the bucket with small strings.

Reply:
Done